service/project.go

```diff
@@ -236,6 +238,15 @@ func (uis *UIServer) modifyProject(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if projectVars.GithubHookID != 0 && projectRef.TracksPushEvents &&
+		responseRef.ForceRepotrackerRun {
+		job := units.NewRepotrackerJob(util.RandomString(), projectRef.Identifier)
```

Comment:
can we have a better identifier here? 

there's also jobs.GetNumber() which provides a monotonically increasing job id, which might be useful.

Reply:
done